src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/Drawing/Drawing.tsx

```diff
@@ -0,0 +1,93 @@
+import { useFormContext } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import { Uploads } from 'modules/Builder/components';
+import { Uploader, UploaderUiType } from 'shared/components';
+import { ItemInputTypes } from 'shared/types';
+import { byteFormatter } from 'shared/utils';
+import { MAX_FILE_SIZE_2MB } from 'shared/consts';
+import { StyledFlexTopCenter, theme } from 'shared/styles';
+
+import { useOptionalItemSetup } from '../../ItemConfiguration.hooks';
+import { DrawingProps } from './Drawing.types';
+import { SharedToggleItemProps, ToggleItem } from '../ToggleItem';
+
+export const Drawing = ({ drawerImage, drawerBgImage }: DrawingProps) => {
+  const { t } = useTranslation();
+  const { watch, setValue } = useFormContext();
+  const mockedTooltipTitle = 'Image Description';
+
+  const commonUploaderProps = {
+    width: 20,
+    height: 20,
+  };
+
+  const uploads = [
+    {
+      title: t('drawingExample'),
+      tooltipTitle: mockedTooltipTitle,
+      upload: (
+        <Uploader
+          {...commonUploaderProps}
+          setValue={(val: string) => setValue(drawerImage, val)}
+          getValue={() => watch(drawerImage)}
+          description={t('uploadImg', { size: byteFormatter(MAX_FILE_SIZE_2MB) })}
+        />
+      ),
+    },
+    {
+      title: t('drawingBackground'),
+      tooltipTitle: mockedTooltipTitle,
+      upload: (
+        <Uploader
+          {...commonUploaderProps}
+          setValue={(val: string) => setValue(drawerBgImage, val)}
+          getValue={() => watch(drawerBgImage)}
+          description={t('uploadImg', { size: byteFormatter(MAX_FILE_SIZE_2MB) })}
+        />
+      ),
+    },
+  ];
+
+  useOptionalItemSetup({
+    itemType: ItemInputTypes.Drawing,
+    name: drawerImage,
+  });
+
+  useOptionalItemSetup({
+    itemType: ItemInputTypes.Drawing,
+    name: drawerBgImage,
+  });
+
+  const HeaderContent = ({ open }: SharedToggleItemProps) =>
+    open ? null : (
+      <StyledFlexTopCenter sx={{ mr: theme.spacing(1) }}>
+        <Uploader
```

Comment:
Are you sure that in open state we also need uploaders, maybe here must be just images?
Could you pls consult with BA's?